config/routes.rb

```diff
@@ -10,6 +10,10 @@
 
   get 'login/callback'
 
+  get 'login/revoke'
+
+  post 'login/revoke'
```

Comment:
Do you need both the GET and the POST route?

Reply:
technically I believe the GET route being present helps prevent an issue when the user hits the back arrow and the browser attempts to navigate back to that page. Hitting the back arrow is automatically a GET request so if the user does hit back/forward and hits that route the app will throw an unknown route 500.

Reply:
to be honest, this should probably just be a GET request, since the request isn't sending any data anyways.